src/object_storage/mod.rs

```diff
@@ -38,4 +39,5 @@ pub trait ObjectStorage {
 pub enum Kind {
     S3,
     Spaces,
+    ScalewayOs,
```

Comment:
Why this name? Is this how they call it?

Reply:
Yep, they call it `Object Storage`. If not clear enough, I can change the name to `ScalewayObjectStorage`.

Reply:
Looks a 0 (zero) and not a O

Reply:
I do confirm it's a O :)
![Screenshot 2021-07-30 at 12 15 59](https://user-images.githubusercontent.com/1393795/127638859-e5576b2c-c8ed-4852-84ea-44cb55428406.png)